specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/datafactory.json

```diff
@@ -1770,6 +1770,48 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/cancelpipelinerun/{runId}": {
+      "post": {
+        "operationId": "Factories_CancelPipelineRun",
+        "x-ms-examples": {
+          "Factories_CancelPipelineRun": {
+            "$ref": "./examples/Factories_CancelPipelineRun.json"
+          }
+        },
+        "description": "Cancel pipeline run by its run ID.",
+        "tags": [
+          "pipelineruns"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/factoryName"
+          },
+          {
+            "$ref": "#/parameters/runId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {

```

Comment:
When the user gets an `OK` response, does that mean the run has been cancelled? Or does it mean that we've started the process of cancelling it?

Reply:
Ok means it is fully cancelled.

Reply:
@milantomic6 could you add to description.